sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/RawContainer.java

```diff
@@ -15,7 +16,12 @@ private RawContainer(
             @JsonProperty("image-name") String imageName,
             @JsonProperty("rlimits") WriteOnceLinkedHashMap<String, RawRLimit> rlimits) {
         this.imageName = imageName;
-        this.rlimits = rlimits;
+
+        if (rlimits != null) {
```

Comment:
From what I've seen, the typical pattern for dealing with potentially `null` members in the raw YAML specs has been to check for emptiness at the site of access, e.g. [YAMLToInternalMappers.java#L248](https://github.com/mesosphere/dcos-commons/blob/13165ea10fe67f4cc49ffaf09f95617eebe7a2c7/sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/yaml/YAMLToInternalMappers.java#L248). I'm not married to it, but it'd be good to have the approach be consistent.